docs/latest/dev/README.md

```diff
@@ -129,7 +129,6 @@ There are many other endpoints on the [Envoy admin interface][] that may be help
 [Quickstart]: https://github.com/envoyproxy/gateway/blob/main/docs/user/quickstart.md
 [make]: https://www.gnu.org/software/make/
 [Github Actions]: https://docs.github.com/en/actions
-[workflows]: .github/workflows
```

Comment:
I was able to preserve the link to workflows with `[workflows]: https://github.com/envoyproxy/gateway/tree/main/.github/workflows`